File: multipleInheritence.java
interface Employee {
    void work();
}

interface Student {
    void study();
}

class MultipleInheritance implements Employee, Student {

    public void work() {
        System.out.println("Employee is working.");
    }

    public void study() {
        System.out.println("Student is studying.");
    }

    public void info() {
        System.out.println("I am an intern.");
    }

    public static void main(String[] args) {
        MultipleInheritance i = new MultipleInheritance();
        i.work();    // from Employee
        i.study();   // from Student
        i.info();    // own method
    }
}

File: multipleInheritence.cpp
#include<iostream>
using namespace std;

class Employee{
    public:
    void work(){
        cout<<"Employee is working."<<endl;

    }

};

class Student{
    public:
    void study(){
        cout<<"Student is studying."<<endl;
    }
};

class Intern : public Employee, public Student{
    public:
    void info(){
        cout<<"I am an intern."<<endl;
    }
};
int main(){
    Intern i;
    i.work();
    i.study();
    i.info();

    return 0;
}

